### PR TITLE
Development  

### PR DESCRIPTION
**Pull request by MrObele**
In development stage @talihomz

### CODE REVIEW

**Comment by uzorjchibuzor** [comment]
@talihomz Here is the link to the review guideline `https://docs.google.com/document/d/1JrKsYPS_t_QOTUm927UIk-e1ZiqZeaTfNufq-P0DxCE/edit`

File: style.css [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+}
+
+.form-group button i{
+    margin-right: 5px;
+}
+
+.policy{
+   padding: 0px 26px;
+   text-align: center; 
+   text-align: center;
```
Comment (heyapricot) [inline style.css:134]: Why is this duplicated?

File: style.css [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    font-size: 14px;
+    opacity: 0.8;
+}
+
+header p a{
+   text-decoration: none;
+   color: #1280C8
+}
+
+header p a:hover{
```
Comment (heyapricot) [inline style.css]: What about writing it like `header p a, header p a:hover { text-decoration: none }`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      Already have an Intuit account?
+      <br>
+      <a href="#">I forgot my user ID or Password</a>
+    </p>
+    <button type="button">
+      <i class="fas fa-lock"></i>Sign In</button>
+  </header>
+
+  <div class="title-images">
+    <div class="header-image">
```
Comment (heyapricot) [inline index.html]: Unused class

File: style.css [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+
+form input+span {
+    padding: 5px 0;
+    font-size: 10px;
+    color: #444;
+}
+
+
+.form-group button {
+    padding: 0.8%;
```
Comment (heyapricot) [inline style.css]: Is this doing something? I disabled/enabled it on Firefox and couldn't see a change

File: style.css [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+    color: #8d9096;
+    margin: 0;
+    padding: 0;
+}
+
+.terms{
+    background: #ECEEF1;
+    margin: 32px -30px -30px;
+    padding: 16px;
+    text-align: center;
```
Comment (heyapricot) [inline style.css:170]: Again, played by disabling/enabling this property and nothing changed so I think it's unused

File: style.css [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+h2{
+    color: #4C4C4C;
+    line-height: 1.1;
+    font-size: 24px;
+    text-align: center;
+    margin: 0 auto;
+}
+.form-div p{
+    text-align: center;
+    padding: 0 16px;
```
Comment (heyapricot) [inline style.css:82]: the padding property also seems to be unused

File: style.css [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+footer a {
+    text-decoration: none;
+}
+
+
+a:hover {
+    text-decoration: underline;
+}
+
+.licence a {
```
Comment (heyapricot) [inline style.css:196]: typo in "licence". It's written License

Comment (heyapricot) [reply style.css:196]: You could have used the global <a> to assign the color fo the links. It is not different on any of the sections

File: style.css (hunk 1 above)
Comment (uzorjchibuzor) [reply style.css:134]: I am removing the duplication right away, couldn't believe we missed that.

File: index.html (hunk 3 above)
Comment (uzorjchibuzor) [reply index.html]: removed.

File: style.css (hunk 4 above)
Comment (uzorjchibuzor) [reply style.css]: removed.

File: style.css (hunk 2 above)
Comment (uzorjchibuzor) [reply style.css]: modified

**Comment by arielcamus** [comment]
@uzorjchibuzor remember to request another code review after introducing changes requested by a TSE until you get a 👍 . Here is the form to do it: https://airtable.com/shrVqH0c0PkuLWIgi

File: style.css [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+    box-sizing: border-box;
+}
+
+a{
+    font-weight: 500;
+}
+
+header{
+    text-align: right;
+    padding: 0% 2%;
```
Comment (heyapricot) [inline style.css:14]: You can omit the `%` when the value is `0`

File: style.css [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+}
+
+header p{
+    display: inline-block;
+    margin-top: 0px;
```
Comment (heyapricot) [inline style.css:19]: You can omit the units when the value is `0`

File: style.css [hunk 10]
```diff
@@ -0,0 +1,211 @@
...
+.policy em{
+    line-height: 20px;
+    letter-spacing: .08em;
+}
+
+.password{
+
+    background: url("./images/padlock-open.png");
+    background-position: right -20px center;
+    background-repeat: no-repeat;
```
Comment (heyapricot) [inline style.css:152]: Could use the shorthand instead `background: url("./images/padlock-open.png") no-repeat right -20px center`

File: index.html [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+    </p>
+    <form action="#" method="post">
+      <div class="form-group">
+        <label for="email">Email Address</label>
+        <input type="email" id="email" name="user_email">
+      </div>
+
+      <div class="form-group">
+        <label for="phone">Phone (recommended)</label>
+        <input type="number" id="phone" name="user_phone">
```
Comment (heyapricot) [inline index.html:57]: Type `number` is for quantites. What you want to use is [`tel`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel)

**Review by heyapricot — APPROVED** [review]
#  Well done completing this project. Looks very good.
Please address the comments and after that, you are ready to move on. No need from my side for further review.
Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

**Review by heyapricot — APPROVED** [review]

**Review by uzorjchibuzor — APPROVED** [review]

**Pull request merged**